content/blog/2020-05-19-remote-roundup/index.markdown

```diff
@@ -0,0 +1,115 @@
+
+Follow along with their learners using this [course schedule](https://cal-poly-advanced-r.github.io/STAT-431/#section-4). Week-by-week, Drs. G & B provide a curated sequence of readings, videos, and small practice activities with careful time estimates so you know what you are signing up for.
+
+<img src="bodwin-glanz-advr.png" title="Advanced R weekly schedule" alt="Sample of an advanced R weekly schedule" width="500"/>
+
+Each week also features a lab assignment and extra challenges so you can practice if you have the time. There is also a public [Discord channel](https://discord.com/invite/SxGR4US) so you can learn with others.
+
+## [Data Visualization](https://datavizm20.classes.andrewheiss.com/)
+
+[<img src="heiss-dataviz.png" title="Data Visualization by Dr. Andrew Heiss" alt="Data Visualization by Dr. Andrew Heiss" style="text-align: center" width="500"/>](https://datavizm20.classes.andrewheiss.com/)
+
+Created by [Professor Andrew Heiss](https://www.andrewheiss.com/) at Georgia State University, also an [RStudio certified trainer](https://education.rstudio.com/trainers/people/heiss+andrew/), this website is an asynchronous online course --- with hours of video, interactive code, code examples, and all sorts of neat stuff! From Andrew's course description:
+
+> "Use R, ggplot2, and the principles of graphic design to create beautiful and truthful visualizations of data"
+
+All materials are open source and creative commons licensed, so anyone can adapt and improve it later.
+
+A personal aside: this beautiful website is built using the Hugo Academic theme with the R blogdown package. You can see the source code for the website on [GitHub](https://github.com/andrewheiss/datavizm20.classes.andrewheiss.com). Andrew's website is [another great example](https://share-blogdown.netlify.app/) of a course website that was [shared on short notice](rstd.io/sharing)!
+
+## [Get Started with tidymodels](https://www.tidymodels.org/start/)
+
+[<img src="tidymodels.png" title="tidymodels.org" alt="tidymodels.org" style="text-align: center" width="500"/>](https://www.tidymodels.org/start/)
+
+We recently launched the new [tidymodels.org](https://www.tidymodels.org/) website, and one of the standout learning resources in there (in my humble opinion) is the *[Get Started](https://www.tidymodels.org/start/)* section. This section features five articles, starting with how to create a model and ending with a beginning-to-end modeling case study. While not fully interactive (yet!), you can copy/paste the code to work through the materials locally. Be on the lookout for integrations with [RStudio Cloud](https://rstudio.cloud) and [learnr](https://rstudio.github.io/learnr/) tutorials that will [work within the IDE](https://blog.rstudio.com/2020/02/25/rstudio-1-3-integrated-tutorials/) after our summer intern, [Ezgi Karaesman](https://education.rstudio.com/trainers/people/karaesmen+ezgi/), works her education magic 💫
```

Comment:
instead of "one of the standout learning resources in there (in my humble opinion)" how about "and one of the most popular resources there" ?